src/extractor/edge_based_graph_factory.cpp

```diff
@@ -220,13 +217,16 @@ void EdgeBasedGraphFactory::InsertEdgeBasedNode(const NodeID node_u, const NodeI
         BOOST_ASSERT(forward_data.edge_id != SPECIAL_NODEID ||
                      reverse_data.edge_id != SPECIAL_NODEID);
 
+        m_compressed_edge_container.AddUncompressedEdge(edge_id_1, node_v, forward_data.distance);
```

Comment:
Yes but if we add this after the loop in the CompressedEdgeContainer we make it atomic and don't need to modify here again. Just requires a second loop there.